packages/office-ui-fabric-react/src/components/Stack/Stack.styles.ts

```diff
@@ -12,7 +12,7 @@ const GlobalClassNames = {
   inner: 'ms-Stack-inner'
 };
 
-export const styles: IStackComponent['styles'] = (props, theme): IStackStylesReturnType => {
+export const styles: IStackComponent['styles'] = (props: IStackProps, theme: ITheme): IStackStylesReturnType => {
```

Comment:
Per our conversation I'd really like to avoid doubling up on types for views and styles and understand why this was needed. I think it's cumbersome for devs and harder to maintain.

Reply:
Apparently this were some flags raised because of other ts errors that were happening at the moment. In reality they are unneeded and I have thus removed them.